src/Me/Navigation/Navbar.js

```diff
@@ -23,9 +23,9 @@ const Navbar = () => {
           src={`${process.env.PUBLIC_URL}/codingcoach-logo-192.png`}
           alt="Logo"
         />
-        <MenuItem to="/me/home" icon={IconHome} label="Home" />
+        <MenuItem to="/me" icon={IconHome} label="Home" />
         <MenuItem to="/" icon={IconMentors} label="Mentors" />
-        <Logout onClick={auth.doLogout}>
+        <Logout to={window.location.pathname} onClick={auth.doLogout}>
```

Comment:
It's happening anyway because logout redirects the user to Auth0 and the callback url is the homepage.

Reply:
https://github.com/Coding-Coach/find-a-mentor/pull/754/commits/687c93f326a9693d788fc622236ed9b3d8ce5a91

`to` is a mandatory prop

Reply:
It doesn't make sense :( Maybe we need to re-organize the styles so we don't have to use Link.
We can keep it for now